src/utils/requests/todos.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+interface Data {
+  id: number;
+  title: string;
+  description: string;
+  deadline: Date;
+  user_id: number;
+  state: string;
+  user_name: string
+}
+
+interface Response {
+  data: Array<Data>;
+  status: number;
+  statusText: string;
+  headers: any;
+  config: any;
+  request: XMLHttpRequest;
+}
+
+
+export const GetTodos = () => {
+  const [ todos, setTodos ] = useState<any>([]);
+
+  const fetchData = async () => {
+    const res = await axios.get<Response>('http://localhost:3000/todos');
+    setTodos(res.data);
+  }
+
+  useEffect(() => {
+    fetchData();
+  }, [])
```

Comment:
`todos`? and at the end of the line `;` please

Reply:
If I write `todos` in the brackets, `fetchData` will be called infinitely. But, in other hand, this syntax means that `fetchData` will be called once when component will be rendered, and if, for example, I add todo, new one will not be appear in the table until refreshing the page.